osu!StreamCompanion/Program.cs

```diff
@@ -28,7 +29,13 @@ static void Main(string[] args)
             string appGuid = ((GuidAttribute)Assembly.GetExecutingAssembly().GetCustomAttributes(typeof(GuidAttribute), false).GetValue(0)).Value.ToString();
             string mutexId = string.Format("Global\\{{{0}}}", appGuid);
 
-            string settingsProfileName = GetSettingsProfileNameFromArgs(args);
+            string settingsProfileName = GetSettingsProfileNameFromArgs(args)?.Trim();
+            if(settingsProfileName.IndexOfAny(Path.GetInvalidFileNameChars()) >=0)
```

Comment:
Doesn't boot when profile name is not provided(launched by double-clicking exe)

Reply:
im dumb smh, should be ez fix, will do soon